changelogs/unreleased/bump-libcurl-to-8.10.1.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/build
+
+* Updated libcurl to version 8.10.1.
```

Comment:
Please, add an issue number (see other changelog entries for the format).

Reply:
Added an issue number.